Add IPv6 adoption documentation (#773)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_adopting-the-compute-service.adoc

[NOTE]
This procedure assumes that {compute_service} metadata is deployed on the top level and not on each cell level. If the {OpenStackShort} deployment has a per-cell metadata deployment, adjust the following patch as needed. You cannot run the metadata service in `cell0`.
+
[NOTE]

> If you are using IPv6, remember to change the load balancer IPs to one that is correct in your environment. +
> E.g: `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`

Reviewer: ```suggestion
If you use IPv6, change the load balancer IP to the IP in your environment, for example,
 `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`.
```
